git_hooks/post-checkout

```diff
@@ -2,4 +2,4 @@
 #
 # Writes git revision into git_revision file
 #
-git show-ref refs/heads/master | cut -d " " -f 1  > git_revision
+git rev-parse HEAD | cut -d " " -f 1  > git_revision
```

Comment:
> Why?

Because I haven't fully *(our terminology here)* PR READY'd this yet... I am not sure why you didn't use `--short` in the first place and this `git` method since it's been in there for a really long time e.g. did you want it stripped out and just use `--short` or keep it as is like I put it in with your abstract? 

Travis build **failed** btw and it's quite relevant... I"m unfamiliar with travis so far *(only used mocha)*... but learning as I go along here.